patroni/__init__.py

```diff
@@ -48,13 +48,19 @@ def load_dynamic_configuration(self):
                 logger.warning('Can not get cluster from dcs')
 
     def get_tags(self):
-        return {tag: value for tag, value in self.config.get('tags', {}).items()
+        tags = {tag: value for tag, value in self.config.get('tags', {}).items()
                 if tag not in ('clonefrom', 'nofailover', 'noloadbalance') or value}
+        tags.update(self.ha.get_dynamic_tags())
+        return tags
 
     @property
     def nofailover(self):
         return bool(self.tags.get('nofailover', False))
 
+    @property
+    def nosync(self):
+        return bool(self.tags.get('nosync', False))
```

Comment:
Right. Nosync means that "don't allow this node to become a sync replica" (is behind WAN for example). It's also used to communicate "I don't want to be a sync standby anymore" to the master to avoid commits hanging until master notices that a standby has gone away.